hudi-utilities/src/test/java/org/apache/hudi/utilities/sources/TestJsonKafkaSourcePostProcessor.java

```diff
@@ -48,17 +56,41 @@
 import static org.junit.jupiter.api.Assertions.assertNotEquals;
 import static org.junit.jupiter.api.Assertions.assertNull;
 import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.mock;
 
-public class TestJsonKafkaSourcePostProcessor extends TestJsonKafkaSource {
```

Comment:
By extending `TestJsonKafkaSource` we were running the tests for `TestJsonKafkaSource` twice. I made some changes to this class to avoid that.